test/integration/operator/garden/garden/garden_test.go

```diff
@@ -748,8 +748,10 @@ var _ = Describe("Garden controller tests", func() {
 		patch := client.MergeFrom(garden.DeepCopy())
 		garden.Spec.RuntimeCluster.CertManagement = &operatorv1alpha1.CertManagement{
 			DefaultIssuer: operatorv1alpha1.DefaultIssuer{
-				Email:  "some.user@some-domain.com",
-				Server: "https://acme-staging-v02.api.letsencrypt.org/directory",
+				ACME: &operatorv1alpha1.ACMEIssuer{
+					Email:  "some.user@some-domain.com",
+					Server: "https://acme-staging-v02.api.letsencrypt.org/directory",
+				},
```

Comment:
Does this configuration mean we rely on a proper account registration during e2e tests, i.e. we have a dependency to Let's Encrypt to run them successfully?

Reply:
Yes, there is dependency on the staging Let's Encrypt server. An account is auto-registered during the test. Alternatively, we could create a CA certificate on the fly and use a CA issuer.

Reply:
Even though I like covering the ACME case, we should probably avoid such a dependency due to different reasons, e.g. firewall/network settings, regional restrictions, availability, etc.

Reply:
For the integration test it is NOT needed to switch to a CA issuer, as the managed resources are not reconciled and the issuer is not created/reconciled.
The e2e test makes use of a CA issuer now.